### PR TITLE
CHE-721: Fix docker authentication error log on create workspace

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**

### CODE REVIEW

**Comment by AndrienkoAleksandr** [comment]
Method "getInitialAuthConfig()" is unused, I deleted it. @garagatyi please review this pull request.

File: plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/InitialAuthConfig.java [hunk 1]
```diff
@@ -11,6 +11,7 @@
 package org.eclipse.che.plugin.docker.client;
 
 import org.apache.commons.codec.binary.Base64;
+import org.eclipse.che.commons.annotation.Nullable;
```
Comment (garagatyi) [inline plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/InitialAuthConfig.java]: It is not needed

**Comment by garagatyi** [comment]
LGTM
@skabashnyuk Can you take a look

**Comment by skabashnyuk** [comment]
ok

**Pull request merged**